test/test_search.rb

```diff
@@ -0,0 +1,96 @@
+# ~*~ encoding: utf-8 ~*~
+path = File.join(File.dirname(__FILE__), 'helper')
+require File.expand_path(path)
+
+context 'Wiki search' do
+
+  SEARCH_TEST_PAGE = <<EOF
+Hobbits are small. Hobbits live in Hobbiton.
```

Comment:
Not all hobbits live in Hobbiton, but Bilbo does.

Reply:
I'm afraid I can't correct that without breaking the tests 😄